include/eve/detail/spy.hpp

```diff
@@ -10,21 +10,24 @@
 #include <ostream>
 namespace spy::detail
 {
-  enum class archs  { undefined_  = -1
-                    , x86_ = 10, amd64_ = 11
-                    , ppc_ = 20, arm_ = 30
-                    , wasm_ = 40
-                    };
-  template<archs Arch> struct arch_info
+enum class archs {
```

Comment:
SPY is an external dependencies, reformatting it with EVE clang format is not necessary.
Also, care to put an issue at SPY for RISC-V support ?

Reply:
Good, I'll take care of that